src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -202,20 +202,18 @@ class HaConfigDashboard extends LitElement {
                     ? html`
                         <a class="button" href="/config/repairs">
                           ${this.hass.localize(
-                            "ui.panel.config.repairs.more_repairs",
-                            {
-                              count: totalRepairIssues - repairsIssues.length,
```

Comment:
This will break translations (if you keep the count in, and don't use it in the translation string, it will not break, but does make it inconsistent betweeen languages)